tests/E2E Tests/WebAppUiTests/UiTestHelpers.cs

```diff
@@ -161,7 +166,12 @@ public static Process StartProcessLocally(string testAssemblyLocation, string ap
                 }
             }
 
-            Process? process = Process.Start(processStartInfo);
+            var currentAttempt = 1;
+            Process? process;
+            do
+            {
+                process = Process.Start(processStartInfo);
+            } while (currentAttempt++ <= maxRetries && ProcessIsAlive(process));

```

Comment:
do we want any retry logic?

example
```csharp
do
{
    process = Process.Start(processStartInfo);
    if (process != null && !process.HasExited)
    {
        break;
    }
    Thread.Sleep(1000 * currentAttempt++); // Exponential backoff
} while (currentAttempt <= maxRetries);
```

Reply:
Yes, this retry logic. I like the exponential backoff, added!